src/Framework/AssemblyLoadBuildEventArgs.cs

```diff
@@ -77,7 +78,11 @@ public override string Message
                 if (RawMessage == null)
                 {
                     string? loadingInitiator = LoadingInitiator == null ? null : $" ({LoadingInitiator})";
+#if FEATURE_ASSEMBLYLOADCONTEXT
+                    RawMessage = FormatResourceStringIgnoreCodeAndKeyword("TaskAssemblyLoadedWithAssemblyLoadContext", LoadingContext.ToString(), loadingInitiator, AssemblyName, AssemblyPath, MVID.ToString(), AppDomainDescriptor ?? DefaultAppDomainDescriptor);
+#else
                     RawMessage = FormatResourceStringIgnoreCodeAndKeyword("TaskAssemblyLoaded", LoadingContext.ToString(), loadingInitiator, AssemblyName, AssemblyPath, MVID.ToString(), AppDomainDescriptor ?? DefaultAppDomainDescriptor);
+#endif
```

Comment:
I have a minor preference for `#if`ing only the resource name so this can't change between worlds, but it's not a big deal either way.

Reply:
Sure, that makes sense. I can make that adjustment.